### PR TITLE
Update float.ex description of ceil/2 and floor/2

### LINKED ISSUE #13083: Float documentation incorrect
### Elixir and Erlang/OTP versions

Irrelevant

### Operating system

Irrelevant

### Current behavior

ceil(number, precision \\ 0)
Rounds a float to the smallest integer greater than or equal to `num`.

floor(number, precision \\ 0)
Rounds a float to the largest number less than or equal to `num`.

# Other such discrepancies???

### Expected behavior

ceil(number, precision \\ 0)
Rounds a float to the smallest **number** greater than or equal to `number`.

floor(number, precision \\ 0)
Rounds a float to the largest number less than or equal to `number`.

### PR DESCRIPTION
**Pull request by 03juan**
Fixes #13083

### CODE REVIEW

File: lib/elixir/lib/float.ex [hunk 1]
```diff
@@ -198,7 +198,7 @@ defmodule Float do
   defp add_dot(acc, false), do: acc <> ".0"
 
   @doc """
-  Rounds a float to the largest number less than or equal to `num`.
+  Rounds a float to the largest float less than or equal to `number`.
```
Comment (03juan) [inline lib/elixir/lib/float.ex:201]:
```suggestion
  Rounds a float to the largest number less than or equal to `number`.
```

File: lib/elixir/lib/float.ex [hunk 2]
```diff
@@ -246,7 +246,7 @@ defmodule Float do
   end
 
   @doc """
-  Rounds a float to the smallest integer greater than or equal to `num`.
+  Rounds a float to the smallest float greater than or equal to `number`.
```
Comment (03juan) [inline lib/elixir/lib/float.ex:249]:
```suggestion
  Rounds a float to the smallest number greater than or equal to `number`.
```

**Review by 03juan — COMMENTED** [review]
Alternatively refer to generic "number", but "float" seems more appropriate so the developer doesn't have to read to the last line above the example to learn "This function always returns floats."

File: lib/elixir/lib/float.ex (hunk 1 above)
Comment (whatyouhide) [reply lib/elixir/lib/float.ex:201]: This returns a float, so calling this "float" is the right way to go IMO.

**Review by whatyouhide — APPROVED** [review]
💯

File: lib/elixir/lib/float.ex (hunk 1 above)
Comment (v0idpwn) [reply lib/elixir/lib/float.ex:201]: The statement isn't true, though. The biggest float smaller than 13.1 isn't 13.0

**Pull request merged**